.github/workflows/rust.yml

```diff
@@ -43,7 +41,7 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: fmt
-          args: --all
+          args: --all -- --check
```

Comment:
Can you please add so that we run it on the crates excluded from the workspace?

Reply:
Done